extensions/jwt/src/lib.rs

```diff
@@ -63,19 +64,14 @@ impl Authenticator for Jwt {
             }
         }
 
-        let (_header, jwt_compact::Claims { custom, .. }) = token.into_parts();
-
-        let CustomClaims {
-            issuer,
-            other: mut claims,
-            ..
-        } = custom;
-
-        // We might want to add the rest later if asked for,
-        // but 'iss' is the only one that I can think of that might be useful.
-        claims.insert("iss".to_string(), issuer.into());
-
-        Ok(Token::new(claims))
+        // We have validated this token, so we're good. We're just re-using the serialized claims data as is.
+        let mut parts = token_str.split('.');
+        parts.next().expect("headers");
+        Ok(Token::from_bytes(
+            base64::engine::general_purpose::URL_SAFE_NO_PAD
+                .decode(parts.next().expect("body"))
+                .unwrap(),
+        ))
```

Comment:
Maybe clean up this a bit. The jwt extension is an example for our users how to build an authentication extension. So the general style what we had before was much nicer than what we have here. Would not nag about it if this wouldn't be something the users implementing auth extensions would read.

Reply:
fair enough, done